homeassistant/components/flipr/sensor.py

```diff
@@ -19,9 +19,10 @@
     SensorEntityDescription(
         key="chlorine",
         name="Chlorine",
-        native_unit_of_measurement=ELECTRIC_POTENTIAL_MILLIVOLT,
+        native_unit_of_measurement=UnitOfElectricPotential.MILLIVOLT,
         icon="mdi:pool",
         state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.VOLTAGE,
```

Comment:
Is it correct to use VOLTAGE for the device_class in the context of Chlorine and Red OX?

Reply:
Yes Chlorine and Red OX are in milli volts.
The PH sensor entity has no unit so an improvement could be to create a unit for it.

I have read the changed files and it is ok for me for flipr integration, no problem.